db/migrate/20220816144252_add_new_event_mail_option_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddNewEventMailOptionToUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :new_event_mail, :boolean, default: true
```

Comment:
Default zou ik false houden. Dan kun je in de mededelingen chat zeggen dat de functionaliteit er is en mensen het zelf aan kunnen zetten. Voor de luie mens onder ons verandert de status quo dan niet.

Reply:
Yes prima hoor